ToolkitApp/tabs/src/components/sample/DocumentationLinks.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { GRAPH_EXPLORER_URL, GRAPH_EXPLORER_DOCS_URL, OFFICIAL_RSC_URL} from "./TabConstants";
+import { Button, Flex } from '@fluentui/react-northstar';
+
+const DocumentationLinks = () => {
+    return (
+        <Flex gap="gap.large">
+            <Button content="Go to Graph Explorer" onClick={() => window.open(GRAPH_EXPLORER_URL)} primary />
+            <Button content="Graph Explorer Documentation" onClick={() => window.open(GRAPH_EXPLORER_DOCS_URL)} primary />
+            <Button content="Resource-Specific Consent" onClick={() => window.open(OFFICIAL_RSC_URL)} primary />
```

Comment:
we should put these in a constants file later

Reply:
e6d71373067f4da9e77eb9f5df034ab201762126 Added localization support, which involves moving these strings to a constants file. Tested it in English and "Chinese".